src/Symfony/Component/PasswordHasher/Tests/Hasher/NativePasswordHasherTest.php

```diff
@@ -119,4 +120,40 @@ public function testNeedsRehash()
         $hasher = new NativePasswordHasher(5, 11000, 5);
         $this->assertTrue($hasher->needsRehash($hash));
     }
+
+    public function testLowOpsLimitsThrows()
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessageMatches('/\$opsLimit must be (\d+) or greater\./');
+
+        new NativePasswordHasher(2);
+    }
+
+    public function testLowMemLimitThrows()
+    {
+        $this->expectException(\InvalidArgumentException::class);
+        $this->expectExceptionMessageMatches('/\$memLimit must be (\d+)(.?) or greater\./');
+
+        new NativePasswordHasher(3, 9999);
+    }
+
+    /**
+     * @testWith [1]
```

Comment:
Not sure we use this in Symfony, I prefer to have real data provider methods

Reply:
We do, here for example:

https://github.com/symfony/symfony/blob/eab28dd5af4825c09bcc8facc9b947acb3b27d4a/src/Symfony/Bundle/FrameworkBundle/Tests/Functional/ConfigDumpReferenceCommandTest.php#L30-L26